src/test/java/org/jenkinsci/plugins/oic/ConfigurationAsCodeTest.java

```diff
@@ -106,39 +102,38 @@ public void testMinimal() throws Exception {
 
         assertTrue(realm instanceof OicSecurityRealm);
         OicSecurityRealm oicSecurityRealm = (OicSecurityRealm) realm;
+        OicServerManualConfiguration serverConf =
+                (OicServerManualConfiguration) oicSecurityRealm.getServerConfiguration();
 
-        assertEquals(
-                "http://localhost/authorize",
-                oicSecurityRealm.getServerConfiguration().getAuthorizationServerUrl());
+        assertEquals("http://localhost/authorize", serverConf.getAuthorizationServerUrl());
+        assertEquals("http://localhost/", serverConf.getIssuer());
         assertEquals("clientId", oicSecurityRealm.getClientId());
         assertEquals("clientSecret", Secret.toString(oicSecurityRealm.getClientSecret()));
         assertFalse(oicSecurityRealm.isDisableSslVerification());
         assertNull(oicSecurityRealm.getEmailFieldName());
         assertFalse(oicSecurityRealm.isEscapeHatchEnabled());
         assertNull(oicSecurityRealm.getFullNameFieldName());
         assertNull(oicSecurityRealm.getGroupsFieldName());
-        assertEquals("openid email", oicSecurityRealm.getServerConfiguration().getScopes());
-        assertEquals(
-                "http://localhost/token",
-                oicSecurityRealm.getServerConfiguration().getTokenServerUrl());
-        assertEquals(
-                TokenAuthMethod.client_secret_post,
-                oicSecurityRealm.getServerConfiguration().getTokenAuthMethod());
+        assertEquals("openid email", serverConf.getScopes());
+        assertEquals("http://localhost/token", serverConf.getTokenServerUrl());
+        assertEquals(TokenAuthMethod.client_secret_post, serverConf.getTokenAuthMethod());
         assertEquals("sub", oicSecurityRealm.getUserNameField());
         assertTrue(oicSecurityRealm.isLogoutFromOpenidProvider());
         assertFalse(oicSecurityRealm.isRootURLFromRequest());
-        assertEquals(null, oicSecurityRealm.getServerConfiguration().getJwksServerUrl());
+        assertEquals(null, serverConf.getJwksServerUrl());
         assertFalse(oicSecurityRealm.isDisableTokenVerification());
     }
 
     @Rule(order = 0)
     public final WellKnownMockRule wellKnownMockRule = new WellKnownMockRule(
             "MOCK_PORT",
-            "{\"authorization_endpoint\": \"http://localhost:%1$d/authorize\","
+            "{\"issuer\": \"http://localhost:%1$d/\","
+                    + "\"authorization_endpoint\": \"http://localhost:%1$d/authorize\","
                     + "\"token_endpoint\":\"http://localhost:%1$d/token\","
                     + "\"userinfo_endpoint\":\"http://localhost:%1$d/user\","
                     + "\"jwks_uri\":\"http://localhost:%1$d/jwks\","
-                    + "\"scopes_supported\": null,"
```

Comment:
the OpenID spec says if something is empty is it ommitted completely and does not contain "null"
may as well say we support the default.